samples/cpp/tutorial_code/HighGUI/BasicLinearTransformsTrackbar.cpp

```diff
@@ -5,8 +5,7 @@
  * @author OpenCV team
  */
 
-#include "opencv2/imgcodecs.hpp"
```

Comment:
Is there a reason why you are removing imgcodecs? Imgproc.hpp is not implicitly including it.